include/pagmo/population.hpp

```diff
@@ -170,6 +170,7 @@ class population
         m_x.reserve(m_x.size() + 1u);
         m_f.reserve(m_f.size() + 1u);
         // The rest is noexcept.
+        update_champion(x, f);
```

Comment:
This function is not ``noexcept`` (see further comments below). It should be made noexcept and then moved above the comment.